lib/constants/scrollAnimation/button_scroll.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+mixin ScrollControllerMixin {
+  ScrollController scrollController = ScrollController();
```

Comment:
Yeah, there is an issue, if I use this, then There is an error in the Blog Display Section

Reply:
Yeah, I have converted this into a statefull widget but it causing error on multiple files, The error is 

The class `_DisplayBlogState` can't implement both `Statr<DisplayBlog>` and `State<StataefullWidget>` because the type argument are different. 

In other file the state changes but the error is there, So I can't convert the Scroll Controller into the Stata of Statefull Widget.